src/NuGet.Core/NuGet.Commands/RestoreCommand/MaccatalystFallback.cs

```diff
@@ -1,11 +1,46 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+using System.Collections.Generic;
+
+using NuGet.Client;
+using NuGet.Frameworks;
+
 namespace NuGet.Commands
 {
     internal class MaccatalystFallback
     {
         internal bool _usedXamarinIOs { get; set; } = false;
+
+        internal static MaccatalystFallback FallbackIfNeeded(NuGetFramework framework)
+
```

Comment:
empty line. 
Surprised our formatter didn't catch this.

Reply:
lol I can't believe _I_ didn't catch this.